src/sentry/api/endpoints/organization_metrics.py

```diff
@@ -160,7 +160,10 @@ class OrganizationMetricsDataEndpoint(OrganizationEndpoint):
     Based on `OrganizationSessionsEndpoint`.
     """
 
+    # Number of groups returned for each page (applies to old endpoint).
     default_per_page = 50
+    # Number of groups returned (applies to new endpoint).
+    default_limit = 20
```

Comment:
The limit was determined after some rough calculations and might be changed in the future.